src/pages/LoginHandler/index.js

```diff
@@ -7,140 +7,141 @@ import {
-              name="horizontal-form-name"
-              onChange={this.handleUserNameInputChange}
-            />
-          </FormGroup>
-          <FormGroup label="Password" isRequired fieldId="horizontal-form-password">
-            <TextInput
-              isRequired
-              type="password"
-              id="horizontal-form-password"
-              name="horizontal-form-password"
-              onChange={this.handlePassWordInputChange}
-            />
-          </FormGroup>
-          <FormGroup fieldId="remember-me">
-            <Checkbox
-              label="Keep me logged in"
-              id="alt-form-checkbox-1"
-              name="alt-form-checkbox-1"
-              className={styles.check}
-            />
-          </FormGroup>
-          <ActionGroup>
-            <Button
-              isBlock
-              variant={variantVal}
-              onClick={() => this.handleLoginSubmit()}
-              className={styles.btn}
-              id="submitBtn"
-              {...(btnDisabled === 'true' ? { isDisabled: 'true' } : {})}
-            >
-              <Title headingLevel="h4" size="xl" style={{ color: btnColor }}>
-                Submit
-              </Title>
-            </Button>
-          </ActionGroup>
-        </Form>
-      </div>
-    );
-    return <AuthLayout toPreview={form} heading="Log into your Pbench Acount" backOpt="true" />;
-  }
-}
+  const form = (
+    <Form className={styles.section}>
+      {alertOpen ? (
+        <AlertGroup isToast>
+          <Alert
+            title={alertComponent.message}
+            variant={alertComponent.variant}
+            action={<AlertActionCloseButton onClose={() => setAlertOpen(!alertOpen)} />}
+          />
+        </AlertGroup>
+      ) : (
+        <></>
+      )}
+      <FormGroup label="Email address" isRequired fieldId="horizontal-form-name">
+        <TextInput
+          isRequired
+          type="text"
+          id="horizontal-form-name"
+          aria-describedby="horizontal-form-name-helper"
+          name="horizontal-form-name"
+          onChange={val => setUsername(val)}
+        />
+      </FormGroup>
+      <FormGroup label="Password" isRequired fieldId="horizontal-form-password">
+        <TextInput
+          isRequired
+          type="password"
+          id="horizontal-form-password"
+          name="horizontal-form-password"
+          onChange={val => setPassword(val)}
+        />
+      </FormGroup>
+      <FormGroup fieldId="remember-me">
+        <Checkbox
+          label="Keep me logged in"
+          id="alt-form-checkbox-1"
+          name="alt-form-checkbox-1"
+          className={styles.check}
+        />
+      </FormGroup>
+      <ActionGroup>
+        <Button
+          isBlock
+          // variant={variantVal}
+          onClick={handleLoginSubmit}
+          className={styles.btn}
+          id="submitBtn"
+          {...(btnDisabled ? { isDisabled: true } : { isDisabled: false })}
```

Comment:
What if we refactored this in order to stay consistent with our ternary operator syntax across the codebase?
```
<Button
...other props...
isDisabled={btnDisabled ? true : false}
/>
```